XML/Other/ReplaceableTestParameters.xml

```diff
@@ -512,7 +512,7 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>DM_EATMEMORY</ReplaceThis>
-		<ReplaceWith>https://github.com/julman99/eatmemory.git</ReplaceWith>
+		<ReplaceWith>https://github.com/LiliDeng/eatmemory.git</ReplaceWith>
```

Comment:
We can't do this. LIS should take a copy and use it.

Reply:
julman99 this is also a personal account, and I need update the tool to use it in our test, do we have a common github account, then can store some tools under that account?